### DISCUSSION TITLE
Auto-add label(s) on import

### DISCUSSION THREAD

**Discussion by stephenmetzger** [photoprism/photoprism#2091] [Q&A] [answered]
Is it possible to have photoprism automatically add a label (not based on object recognition) to all imported photos? I will have multiple people adding photos to the photo subdirectory and it would be nice to add a "needs attention" label to all photos on import. This would then make it easy to open the photos with that label to make any needed edits on additional labels, date/time/location corrections, etc.

Alternatively, could all new photos go into the "Review" tab. I'm not exactly sure what criteria needs to be met for photos to go to the Review tab. 

Thanks.

**Comment by graciousgrey** [comment, marked as answer]
Are you using the  import or index method (https://docs.photoprism.app/user-guide/library/)?
How will everyone add the photos, using the [web upload](https://docs.photoprism.app/user-guide/library/upload/) (import) or using [webdav](https://docs.photoprism.app/user-guide/library/webdav/)?

When using the web-upload everyone could upload the photos into an album e.g. "needs-editing". 

The review section cannot be customized at the moment, it is filled with photos that have a low quality score (see https://docs.photoprism.app/user-guide/organize/review/). E.g. screenshots or photos send to you via messengers.

So it's not possible that all new photos go into review. What you can do is sorting all photos by "recently added".
